Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN apt-get update -y && \
       libffi-dev \
       openssl \
       python-pip \
+      libcrack2 \
```

Comment:
Add libcrack2 program to provisioner image (for possibility to do a "preflight" testing of password strength)